superbench/tools/system_info.py

```
@@ -140,7 +145,7 @@ def parse_table_lines(self, lines, key):
                             dict[key[i]] = line[index[i]]
                     list.append(dict)
         except Exception:
-            print('Error: key error in __parse_table_lines')
+            print('Error: key error in parse_table_lines')
```

could you raise it instead of print?
```py
except Exception as ex:
    raise RuntimeError('...') from ex
```

---

updated